src/main/resources/assets/farmersdelight/lang/fi_fi.json

```diff
@@ -134,29 +162,38 @@
   "item.farmersdelight.stuffed_pumpkin": "Lautasellinen täytettyä kurpitsaa",
   "block.farmersdelight.honey_glazed_ham_block": "Hunajakuorrutettu kinkku",
   "item.farmersdelight.honey_glazed_ham": "Lautasellinen hunajakuorrutettua kinkkua",
+  "block.farmersdelight.honey_glazed_ham_block": "Lammaspaimenen piiras",
```

Comment:
This line seems to have the wrong ID (duplicated from `honey_glazed_ham`, seems to be `shepherds_pie`).
It might error when loading the lang.